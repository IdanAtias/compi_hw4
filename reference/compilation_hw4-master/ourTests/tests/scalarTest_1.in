int d = 4;
int x = - 5, y = 5, z = d;

d * d * d;
d / d / d;
d + 8;
-3;
- -4;
x * ((y - x) + z) / d;
x * y - x + z / d;

-5 * ((y - x) + 4) / d;
x * 5 - - 5 + z / 4;

4 * 4 * d;
d / 4 / 4;
4 + 8;
-3;
- -4;

3 + 5 - -2 * 15 / 2 + 4 - 9;
(2-3)*4+7+(6/2)* -2;
