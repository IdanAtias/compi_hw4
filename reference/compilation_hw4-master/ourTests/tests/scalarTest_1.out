d=
4
x=
-5
y=
5
z=
4
64
0
12
-3
4
-17
-19
-17
-19
64
0
12
-3
4
18
-3
